product_inventory_service/conf/application.conf

```diff
@@ -9,11 +9,18 @@ slick.dbs.default.db.driver="org.postgresql.Driver"
 play.evolutions.db.default.autoApply=true
 
 # Assign default db credentials
-slick.dbs.default.db.url="jdbc:postgresql://localhost:8080/products"
+slick.dbs.default.db.url="jdbc:postgresql://localhost:5432/products"
 slick.dbs.default.db.user = "postgres"
 slick.dbs.default.db.password = "postgres"
 
 # Try to assign db credentials through environment variables
 slick.dbs.default.db.url="jdbc:postgresql://"${?POSTGRES_URL_PRODUCTS}
 slick.dbs.default.db.user=${?POSTGRES_USER}
 slick.dbs.default.db.password=${?POSTGRES_PASSWORD}
+
+# Vault config
+vault.api.path="http://localhost:8200/v1"
+vault.api.path=${?VAULT_API_PATH}
+vault.api.totp.keyPrefix="products__"
+vault.auth.credentials.initFile="./.dependencies/vault/credentials.json"
```

Comment:
If you plan on building an identity verification system for requests to the product inventory, that's awesome. But you will also still need to be able to generate TOTP codes with the "finance__" key prefix to access the transfer API in user management.